version.txt

```diff
@@ -1 +1 @@
-1.0.0.M2
\ No newline at end of file
+1.0.1.M2
```

Comment:
this should still be 1.0.0.M2 
since we are trying to release 1.0.0
To satisfy your need lets do 1.0.0.M2-1